OpenOversight/app/static/js/find_officer.js

```diff
@@ -37,9 +37,14 @@ $(document).ready(function() {
             data: {department_id: dept_id}
         }).done(function(ranks) {
             $('input#rank').replaceWith('<select class="form-control" id="rank" name="rank">');
+            var rank_box = $('select#rank')
+            // Add the null case first
+            rank_box.append(
+                $('<option></option>').attr("value", "Not Sure").text("Not Sure")
+            );
```

Comment:
We don't really need to separate it into separate methods, but up to you what makes most sense.

I'd suggest not even using jQuery but seeing as it's already there and probably not easily removed (considering it's a dependency of Bootstrap 3 anyway) then might as well if the API makes sense (though I think increasingly fewer people actually know how to use jQuery these days! I definitely don't remember almost anything about it any more 😂)

```suggestion
            rank_box.append(
                $('<option value="Not Sure">Not Sure</option>')
            );
```

Reply:
Oh this is a great change, thanks! Honestly most of the JS stuff I'm just copying and adapting, very glad to have more input on how to make that better :smile: